README.md

```diff
@@ -1 +1,2 @@
-# django-girls-blog-DrEdi
\ No newline at end of file
+[![Build Status](https://travis-ci.org/kpi-web-guild/django-girls-blog-DrEdi.svg?branch=feature%2Ftravis-ci-integration)](https://travis-ci.org/kpi-web-guild/django-girls-blog-DrEdi)
```

Comment:
Do not really understand what's wrong.
I had to do this from master, right?
But when I just wanted to fix it(add it in master, after come to `feature/travis-ci-integration` and push it git said that 'Everything is up-to-date'